blueprints/acceptance-test/files/tests/acceptance/__name__-test.js

```diff
@@ -17,14 +17,14 @@ describe('Acceptance: <%= classifiedModuleName %>', function() {
   });
 
   afterEach(function() {
-    Ember.run(application, 'destroy');
+    application.destroy();
   });
 
-  it('can visit /<%= dasherizedModuleName %>', function() {
+  beforeEach(function() {
```

Comment:
It's fairly common to separate setup concerns. In this case, application setup vs page navigation, but what I'd probably do IRL, is either extract the app setup into its own helper, or If I was feeling like a BDD purist, I'd introduce a new description coupled with the new setup. E.g.

```